google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStoragePath.java

```diff
@@ -41,9 +41,17 @@
 import javax.annotation.concurrent.Immutable;
 
 /**
- * Google Cloud Storage {@link Path}.
+ * A Google Cloud Storage specific implementation of the {@code java.nio.file.Path} interface. An
+ * instances of this class may be used to locate an object or a "pseudo-directory" in the Storage.
```

Comment:
instances --> instance
may be used to locate an object --> locates an object
the Storage --> GCS